spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -55,6 +55,92 @@ def it_redirects_to_user_login
     end
   end
 
+  describe "create" do
+    let(:work) { create(:work) }
+
+    context "when user is not logged in" do
+      it "redirects to login" do
+        post :create, params: { work_id: work.id }
+        it_redirects_to_user_login
+      end
+    end
+
+    context "when user is logged in" do
+      let(:user) { create(:user) }
+      let(:pseud) { user.default_pseud }
+
+      before do
+        fake_login_known_user(user)
+      end
+
+      context "when pseud doesn't belong to the user" do
+        let(:otheruser) { create(:user) }
```

Comment:
Could you use an underscore to separate the words in variable names, e.g. `other_user`?